pygmt/src/rose.py

```diff
@@ -63,10 +63,11 @@ def rose(self, data=None, length=None, azimuth=None, **kwargs):
     data : str or {table-like}
         Pass in either a file name to an ASCII data table, a 2D
         {table-classes}.
-        Use option ``columns`` to choose which columns are length and azimuth,
-        respectively. If a file with only azimuths is given, use ``columns`` to
-        indicate the single column with azimuths; then all lengths are set to
-        unity (see ``scale = 'u'`` to set actual lengths to unity as well).
+        Use parameter ``incols`` to choose which columns are length and
+        azimuth, respectively. If a file with only azimuths is given, use
+        ``incols`` to indicate the single column with azimuths; then all
+        lengths are set to unity (see ``scale = 'u'`` to set actual
+        lengths to unity as well).
```

Comment:
```suggestion
        lengths are set to unity (see ``scale="u"`` to set actual
        lengths to unity as well).
```

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/2090/commits/069fa4c81779bf9cef2c3b2921997d8177969ac9 and https://github.com/GenericMappingTools/pygmt/pull/2090/commits/dd104ce02a8c80423e10008a3e8224ed04ab1824.